recipes/libacl/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+class LibAclConan(ConanFile):
+    name = "libacl"
+    description = "Commands for Manipulating POSIX Access Control Lists"
+    topics = ("conan", "acl", "POSIX")
+    license = "GPL-2.0-or-later"
+    homepage = "https://savannah.nongnu.org/projects/acl/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+    requires = ["libattr/2.5.1"]
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _doc_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "doc"
+        )
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("libacl is just supported for Linux")
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True,
+                  destination=self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(
+            self, win_bash=tools.os_info.is_windows)
```

Comment:
```suggestion
        self._autotools = AutoToolsBuildEnvironment(self)
```

if windows is not supported, it's useless to add win_bash